### PR TITLE
WT-7549 change uses of logid to objectid, log_structured to has_objects.

### PR DESCRIPTION
**Pull request by ddanderson**

### CODE REVIEW

**Comment by ddanderson** [comment]
@sueloverso and @keitharnoldsmith, this is mostly a mechanical change.  If it helps, the first commit of this change is entirely a mechanical substitution plus a couple nearby comments modified.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM

File: src/block/block_ckpt_scan.c [hunk 1]
```diff
@@ -220,15 +220,15 @@ __wt_block_checkpoint_last(WT_SESSION_IMPL *session, WT_BLOCK *block, char **met
...
+    uint32_t checksum, objectid, size;
     const uint8_t *p, *t;
     bool found;
 
     *metadatap = *checkpoint_listp = NULL;
     WT_RET(__wt_buf_init(session, checkpoint, WT_BLOCK_CHECKPOINT_BUFFER));
 
-    /* TODO: scan all log IDs. */
-    logid = 0;
+    /* TODO: tiered: scan all log IDs. */
```
Comment (sueloverso) [inline src/block/block_ckpt_scan.c:230]: Very minor: "scan all object IDs"

**Review by sueloverso — APPROVED** [review]
LGTM: I have one tiny comment on a TODO comment so ignore that if you want.

**Comment by ddanderson** [comment]
I'm going to merge this.  I had a successful build, then changed a comment, then an (unrelated) failure on macos, and restarting the failed task didn't seem to have any effect.

**Pull request merged**